tpm2/constants.go

```diff
@@ -273,3 +273,39 @@ var hashConstructors = map[Algorithm]func() hash.Hash{
 	AlgSHA384: sha512.New384,
 	AlgSHA512: sha512.New,
 }
+
+// NV attributes
```

Comment:
Are these used for anything?

Reply:
I'm using those when creating the index. Like so:

```go
	if err := tpm2.NVDefineSpace(rwc,
		tpm2.HandleOwner,
		idx,
		emptyPassword,
		emptyPassword,
		nil,
		tpm2.NVAttrOwnerWrite|tpm2.NVAttrOwnerRead|tpm2.NVAttrAuthRead|tpm2.NVAttrPPRead,
		uint16(len(data)),
	); err != nil {
		panic(err)
	}
```

Reply:
Then let's make a named type for this and use hex values instead of `iota` (like `KeyProp`).
And we should change NVDefineSpace `attributes` argument to use this new type.